src/components/DiseaseTable.tsx

```diff
@@ -103,6 +105,29 @@ const DiseaseTable = ({
         ),
     };
 
+    const handleCellInHoveredRow = (
+        record: UnpackedDiseaseCellLine,
+        index: number | undefined
+    ) => {
+        const hoveredAndDataComplete =
+            index === hoveredRowIndex && record.status === "data complete";
+        return {
+            className: hoveredAndDataComplete ? hoveredRow : "",
+            onMouseEnter: () => {
+                if (index !== undefined) {
```

Comment:
I would move this to an early return 
`if (index === undefined) {
            return {};
        }` 
because there isn't a meaningful state that you're checking for here, it's just a typescript issue